src/mmw/apps/modeling/models.py

```diff
@@ -131,6 +152,20 @@ class Meta:
         auto_now_add=True)
     modified_at = models.DateTimeField(
         auto_now=True)
+    weather_type = models.CharField(
+        max_length=255,
+        default=WeatherType.DEFAULT,
```

Comment:
We set it to `DEFAULT` for all scenarios. Realistically, it'll only be checked for GWLF-E scenarios, never for TR-55. It felt cleaner to have it be `DEFAULT` for everyone, then have to support `NULL`s just for better TR-55 semantics.